harvester/Exporter.py

```diff
@@ -111,12 +70,15 @@ def _generate_gmeta(self, export_filepath, temp_filepath, only_new_records, star
 
             record = (dict(zip(
                 ['record_id', 'title', 'pub_date', 'contact', 'series', 'source_url', 'deleted', 'local_identifier',
-                 'modified_timestamp',
+                 'item_url', 'modified_timestamp',
                  'repository_url', 'repository_name', 'repository_thumbnail', 'item_url_pattern',
                  'last_crawl_timestamp'], row)))
             record["deleted"] = int(record["deleted"])
 
-            record["dc:source"] = self._construct_local_url(record)
+            if record["item_url"] == "":
+                record["item_url"] = self.db._construct_local_url(record)
+
+            record["dc:source"] = record["item_url"]
```

Comment:
I think we can cut this if we are removing dc:source entirely

Reply:
We can't change the fields in the gmeta export without doing some associated work on the discovery UI.

Reply:
Oh right, that makes sense! I think there will be a few other changes like this once we work on geospatial metadata, so let's table this for now.